### PR TITLE
Present user with ability to go to the map when selecting a space.

### PR DESCRIPTION
**Pull request by kberg**
Fixes #2584

image
image

### CODE REVIEW

File: src/client/components/PlayerHome.vue [hunk 1]
```diff
@@ -373,24 +373,24 @@ export default Vue.extend({
     navigatePage(event: KeyboardEvent) {
       const inputSource = event.target as Element;
       if (inputSource.nodeName.toLowerCase() !== 'input') {
-        let idSuffix: string | undefined = undefined;
+        let id: string | undefined = undefined;
```
Comment (kberg) [inline src/client/components/PlayerHome.vue:376]: This just simplifies reading the IDs while trying to navigate the code.

File: src/client/components/SelectSpace.vue [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
         :enableDontShowAgainCheckbox="true"
         ref="confirmation"
         v-on:accept="confirmPlacement"
         v-on:dismiss="cancelPlacement"
         v-on:hide="hideDialog" />
-    <div v-if="showtitle" class="wf-select-space">{{ $t(playerinput.title) }}</div>
-    <div v-if="warning" class="nes-container is-rounded"><span class="nes-text is-warning" v-i18n>{{ warning }}</span></div>
+    <div v-if="showtitle" class="wf-select-space">
+      {{ $t(playerinput.title) }}
+      <template v-if="experimental()">&nbsp;&nbsp;<a href="#board">go to map</a></template>
```
Comment (kberg) [inline src/client/components/SelectSpace.vue:12]: I don't like having this repeated. I could very simply add a GoToBoard component. ;)

**Review by kberg — COMMENTED** [review]
I haven't verified that this works with The Moon yet.

File: src/client/components/waitingFor/GoToMap.vue [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  name: 'GoToMap',
+  props: {
+    playerinput: {
+      type: Object as () => PlayerInputModel,
+    },
+  },
+  methods: {
+    onclick(event: MouseEvent) {
+      const id = isMarsSpace(this.playerinput.availableSpaces?.[0] ?? '') ? 'shortkey-board' : 'shortkey-moonBoard';
+      console.log(id);
```
Comment (bafolts) [inline src/client/components/waitingFor/GoToMap.vue:22]: Would remove before merging.

```suggestion
```

**Review by bafolts — APPROVED** [review]

File: src/client/components/waitingFor/GoToMap.vue (hunk 3 above)
Comment (kberg) [reply src/client/components/waitingFor/GoToMap.vue]: > Would remove before merging.

Ah yes, thanks.

File: src/client/components/SelectSpace.vue (hunk 2 above)
Comment (kberg) [reply src/client/components/SelectSpace.vue]: (Which is what I did.)

**Pull request merged**